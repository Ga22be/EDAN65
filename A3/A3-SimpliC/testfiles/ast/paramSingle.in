int a(int house) {
    int c;
    c = 1 * 3 / 2 + house;
}