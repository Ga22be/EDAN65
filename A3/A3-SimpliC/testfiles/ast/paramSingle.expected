Program
  List
    Function
      IdDecl ID="a"
      List
        Parameter
          IdDecl ID="house"
      List
        Declaration
          IdDecl ID="c"
        Assignment
          IdUse ID="c"
          Add
            Div
              Mul
                Numeral NUMERAL="1"
                Numeral NUMERAL="3"
              Numeral NUMERAL="2"
            IdUse ID="house"
